### PR TITLE
add missing translation for voltage level creation

### PR DESCRIPTION
**Pull request by basseche**

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -1404,5 +1404,6 @@
     "guidancePopUp.action":"Close editor",
     "generateNad": "Generate",
     "networkModificationActivationError": "An error occured while enabling / disabling modification",
-    "AddAutomaton": "Add an automaton"
+    "AddAutomaton": "Add an automaton",
+    "BusBarCountMustBeGreaterThanOrEqualToStartTime" : "Busbar count must be greater or equal to 1"
```
Comment (ghazwarhili) [inline src/translations/en.json:1408]: number of Busbar must be greater than or equal to 1

File: src/components/dialogs/network-modifications/voltage-level/creation/voltage-level-creation-dialog.jsx [hunk 2]
```diff
@@ -104,7 +104,7 @@ const formSchema = yup
                 is: (lowShortCircuitCurrentLimit) => lowShortCircuitCurrentLimit != null,
                 then: (schema) => schema.required(),
             }),
-        [BUS_BAR_COUNT]: yup.number().min(1).nullable().required(),
+        [BUS_BAR_COUNT]: yup.number().min(1, 'BusBarCountMustBeGreaterThanOrEqualToStartTime').nullable().required(),
```
Comment (ghazwarhili) [inline src/components/dialogs/network-modifications/voltage-level/creation/voltage-level-creation-dialog.jsx:107]: ToStartTime  ?

**Review by ghazwarhili — CHANGES_REQUESTED** [review]

File: src/components/dialogs/network-modifications/voltage-level/creation/voltage-level-creation-dialog.jsx (hunk 2 above)
Comment (basseche) [reply src/components/dialogs/network-modifications/voltage-level/creation/voltage-level-creation-dialog.jsx:107]: oups !

File: src/translations/en.json (hunk 1 above)
Comment (basseche) [reply src/translations/en.json]: solved

File: src/translations/en.json [hunk 3]
```diff
@@ -1405,5 +1405,6 @@
     "guidancePopUp.action":"Close editor",
     "generateNad": "Generate",
     "networkModificationActivationError": "An error occured while enabling / disabling modification",
-    "AddAutomaton": "Add an automaton"
+    "AddAutomaton": "Add an automaton",
+    "BusBarCountMustBeGreaterThanOrEqualToOne" : "Number of busbar must be greater or equal to 1"
```
Comment (ghazwarhili) [inline src/translations/en.json:1409]: greater than

File: src/components/dialogs/network-modifications/voltage-level/creation/voltage-level-creation-dialog.jsx [hunk 4]
```diff
@@ -104,7 +104,7 @@ const formSchema = yup
                 is: (lowShortCircuitCurrentLimit) => lowShortCircuitCurrentLimit != null,
                 then: (schema) => schema.required(),
             }),
-        [BUS_BAR_COUNT]: yup.number().min(1).nullable().required(),
+        [BUS_BAR_COUNT]: yup.number().min(1, 'BusBarCountMustBeGreaterThanOrEqualToOne').nullable().required(),
         [SECTION_COUNT]: yup.number().min(1).nullable().required(),
```
Comment (ghazwarhili) [inline src/components/dialogs/network-modifications/voltage-level/creation/voltage-level-creation-dialog.jsx:108]: we could fix the translation for SECTION_COUNT also in this PR

```suggestion
        [SECTION_COUNT]: yup.number().min(1, 'SectionCountMustBeGreaterThanOrEqualToOne').nullable().required(),
```

**Review by ghazwarhili — APPROVED** [review]

**Pull request merged**